coalib/output/ConsoleInteraction.py

```diff
@@ -466,6 +554,27 @@ def print_results_no_input(log_printer,
     """
     for result in result_list:
 
+        console_printer.print('\n'+format_lines(result.message, symbol='!'))
+        if hasattr(section, 'name'):
+            console_printer.print(
+                '**** {bear} [Section: {section} | Severity: '
+                '{severity}] ****'
+                .format(
+                    bear=result.origin,
+                    section=section.name,
+                    severity=RESULT_SEVERITY.__str__(
+                                              result.severity
+                                              )
+                        ),
+                color=RESULT_SEVERITY_COLORS[result.severity])
+        else:   # pragma: no cover
+            console_printer.print(
+                '**** {bear} [Section {section} | Severity '
+                '{severity}] ****'
+                .format(bear=result.origin, section='<empty>',
+                        severity=RESULT_SEVERITY.__str__(result.severity)),
+                color=RESULT_SEVERITY_COLORS[result.severity])
```

Comment:
The code in `else` block looks almost the same as code in `if` block - why not merge them together

Reply:
Fixed it. The code can be merged in `print_results` function as well